tox_core/src/dht/server_ext.rs

```diff
@@ -26,7 +26,7 @@ impl ServerExt for Server {
         socket: UdpSocket,
         mut rx: Receiver<(Packet, SocketAddr)>,
         stats: Stats
-    ) -> Pin<Box<dyn Future<Output = Result<(), Error>> + Send>> {
+    ) -> Pin<Box<dyn Future<Output = Result<(), Error>>>> {
```

Comment:
Can it be async fn?

Reply:
It can't - traits can't have async fns currently.

Reply:
may be try to use async_trait crate?